modules/reloadBidAdapter.js

```diff
@@ -278,13 +279,19 @@ function ReloadClientTool(args) {
       confined: detectConfined(),
       protStr: _getProtocolString(),
 
-      hostDomain: decodeURIComponent(window.location.host),
-      hostPagePath: decodeURIComponent(window.location.pathname),
-      hostPageUrl: decodeURIComponent(window.location.href),
+      hostDomain: decodeURIComponent(locInf.host),
+      hostPagePath: decodeURIComponent(locInf.pathname),
+      hostPageUrl: decodeURIComponent(locInf.href),
       hostPageTitle: document.title,
     };
 
-    function _genWinInfo () {
+    function getBestLocInf() {
```

Comment:
You should using bidderRequest.refererInfo rather than tyring to find the top window yourself
http://prebid.org/dev-docs/bidder-adaptor.html#bidrequest-parameters

Reply:
@mike-chowla  We are aware of that prebid feature, but we use a standard library across all our adapters. Nevertheless, we will implement this on next version -if possible- so that we do not loose one week. We would really appreciate it!

Reply:
@mike-chowla  Hello Mike, we tried to use your suggestion but it is not possible. We use a single module ('ReloadClientTool') that acts as a common interface across all our adapters, so we cannot use features of any specific wrapper within that class. We explained this to Jaimin in relation to our original version (see https://github.com/prebid/Prebid.js/pull/3812).